graphql-language/src/ast.rs

```diff
@@ -283,16 +282,11 @@ pub enum Type<'a> {
 }
 
 /// NamedType : Name
-pub struct NamedType<'a> {
```

Comment:
That's fair, I err on the side of the reference implementation, but it doesn't have to be that way now.